examples/greet-and-react/package.json

```diff
@@ -13,9 +13,9 @@
     "@aoberoi/passport-slack": "^1.0.5",
     "@slack/client": "^4.3.1",
     "@slack/events-api": "^1.0.0",
-    "body-parser": "^1.16.1",
+    "body-parser": "^1.18.3",
```

Comment:
Can't we just remove `body-parser` as a dependency?

Reply:
not in this commit, but we should in #59

Reply:
🐙